src/shared/pager.c

```diff
@@ -89,7 +89,7 @@ int pager_open(bool no_pager, bool jump_to_end) {
 
         /* In the child start the pager */
         if (pager_pid == 0) {
-                const char* less_opts, *less_charset;
+                const char* pager_opts, *pager_charset;
```

Comment:
`less_charset` is used to set LESSCHARSET, and your patch does not change that. So the name should not change.